code/workspaces/web-app/src/components/app/SideBar.js

```diff
@@ -1,87 +1,65 @@
+import SideBarGroup from './SideBarGroup';
+import SideBarButton from './SideBarButton';
 
 const { projectPermissions: { PROJECT_STORAGE_LIST, PROJECT_STACKS_LIST } } = permissionTypes;
 
-const { DISCOURSE, SLACK } = navBarLinks;
-export const drawerWidth = 240;
+const { DISCOURSE } = navBarLinks;
+export const drawerWidth = 200;
 
 const styles = theme => ({
-  header: {
-    ...theme.mixins.toolbar,
-    backgroundColor: theme.palette.secondary[900],
-    display: 'flex',
-    alignItems: 'center',
-  },
-  drawerPaper: {
-    position: 'fixed',
-    height: '100%',
-    width: drawerWidth,
-    backgroundColor: theme.palette.secondary[800],
-    borderWidth: 0,
-  },
-  logo: {
-    width: 140,
-    marginLeft: 16,
-  },
-  drawerList: {
+  itemList: {
     height: '100%',
     display: 'flex',
     flexDirection: 'column',
+    padding: 0,
   },
-  spacer: {
-    flex: 1,
+  sideBar: {
+    height: '100%',
+    width: drawerWidth,
+    borderRight: `1px solid ${theme.palette.divider}`,
+    overflow: 'auto',
+    padding: `0 ${theme.spacing(2)}px`,
   },
 });
 
-const datalabLinks = [
-  { ...DISCOURSE, icon: 'question_answer' },
-  { ...SLACK, icon: 'message' },
-  { displayName: 'Help', href: extendSubdomain('docs'), icon: 'help_outline' },
-];
-
 const SideBar = ({ classes, userPermissions }) => (
-  <Drawer classes={{ paper: classes.drawerPaper }} variant="permanent">
-    <header className={classes.header}>
-      <img className={classes.logo} src={datalabsLogo} alt="datalabs-logo" />
-    </header>
-    <List className={classes.drawerList}>
-      <NavLink to="/" label="Dashboard" icon="dashboard" />
-      <PermissionWrapper userPermissions={userPermissions} permission={PROJECT_STORAGE_LIST}>
-        <SideBarSubheader>Data</SideBarSubheader>
-        <NavLink to="/storage" label="Storage" icon="storage" />
-      </PermissionWrapper>
-      <SideBarSubheader>Analysis</SideBarSubheader>
-      <PermissionWrapper userPermissions={userPermissions} permission={PROJECT_STACKS_LIST}>
-        <NavLink to="/notebooks" label="Notebooks" icon="book" />
-      </PermissionWrapper>
-      <NavLink to="/dask" label="Dask" icon="apps" />
-      <NavLink to="/spark" label="Spark" icon="apps" />
-      <PermissionWrapper userPermissions={userPermissions} permission={PROJECT_STACKS_LIST}>
-        <SideBarSubheader>Publish</SideBarSubheader>
-        <NavLink to="/publishing" label="Sites" icon="web" />
-      </PermissionWrapper>
-      <div className={classes.spacer} />
-      <Divider />
-      {datalabLinks.map(({ displayName, href, icon }) => <NavLink
-          key={`nav-link-${displayName}`}
-          label={displayName}
-          icon={icon}
-          onClick={() => window.open(href)}
-        />)}
+  <div className={classes.sideBar}>
+    <List className={classes.itemList}>
+      <SideBarGroup>
+        <SideBarButton to="/" label="Dashboard" icon="dashboard" />
+      </SideBarGroup>
+
```

Comment:
Wondering why you changed the sidebar ordering of storage/analysis/publish? I thought having the breaks in there made sense.

Reply:
I agree that the breaks make sense but I don't think they really work when there is only one item in a particular category. My thinking was that when there are two or more things that could be grouped together then they should be put into a category, but until then leave them out of a category. Happy to hear if people disagree with this plan!